conferences/2025-agu.md

```diff
@@ -37,3 +37,42 @@ We're excited to hear what you have to say at our session!
 ## Invited speakers
 
 _TBD_
+
+
+## Organizers
+
+
+### Primary Convener: Qiusheng Wu
+
+_TODO: Insert mini-bio_
+
+
+### Convener: Max Jones
+
+_TODO: Insert mini-bio_
```

Comment:
@maxrjones please suggest a bio! Can you please also provide a link or paste a photo into the comments here?